src/main/java/seedu/address/logic/parser/sale/ListCommandParser.java

```diff
@@ -1,12 +1,18 @@
 package seedu.address.logic.parser.sale;
```

Comment:
@hakujitsu parser logic has changed abit to adapt to my sale list subclass as well